### ISSUE TITLE
Uncaught (in promise) Error: The checkoutCreate mutation failed due to an unknown error.

### ISSUE THREAD

**Issue by Jamal1337** [Shopify/js-buy-sdk#866]
I am using this sdk inside a gatsby project but I keep getting the error:

> Uncaught (in promise) Error: The checkoutCreate mutation failed due to an unknown error.

As far as I understood it seems to be an issue with the API credentials, but I know that the ones I provide are 100% correct and worked before. I am a bit clueless how to fix that issue. None of the requests are working, I tried fetching products as well but it does not work either.

Did anybody else experience that issue or has an idea how to solve it? I am initializing the client like that:
`const client = Client.buildClient({
    storefrontAccessToken: password,
    domain: isCustomDomain ? storeUrl : `${storeUrl}.myshopify.com`,
  });`

and then try to create the checkout:
`client.checkout.create().then((checkout) => {
        // Do something with the checkout
      });`

so basically 1:1 the code from the readme.

Any help is really appreciated, thanks!

**Comment by Jamal1337** [comment]
found the issue, nvm
